docs/processes/fuzzer_targets.md

```diff
@@ -0,0 +1,119 @@
+<!--
+© 2019 and later: Unicode, Inc. and others.
+License & terms of use: http://www.unicode.org/copyright.html
+-->
+
+Developing Fuzzer Targets for ICU APIs
+======================================
+
+This documents describes how to develop a [fuzzer](https://opensource.google.com/projects/oss-fuzz) target for an ICU API and its integration into the ICU build process.
+
+### Directory and naming conventions
+
+Fuzzer targets are exclusively in directory [```source/test/fuzzer/```](https://github.com/unicode-org/icu/tree/master/icu4c/source/test/fuzzer) and end with ```_fuzzer.cpp```. Only files with such ending are recognized and executed as fuzzer targets by the OSS-Fuzz system.
+
+### General structure of a fuzzer target
+
+As a minimum, a fuzzer target contains the function
+
+```
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size){
+  ...
+}
+```
+
+This function is expected and invoked by the fuzzer system. The ```data``` parameter contains the fuzzer-controled data of size ```size``` bytes. Part or all of this data is then passed into the ICU API under test.
```

Comment:
Please split lines so that they are not longer than about 100 characters.

Reply:
Done